index.js

```diff
@@ -3,7 +3,7 @@ var bug4 = require('./bugs/bug4');
 var bug6 = require('./bugs/bug6');
 var bug81a = require('./bugs/bug81a');
 
-var doNothingValues = ['none', 'auto', 'content', 'inherit', 'initial', 'unset'];
```

Comment:
Bad practice to declarate rule in index.
These checks chould be repeated in each ./bugs/bunny file!